ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-ref.yaml

```diff
@@ -0,0 +1,10 @@
+ref:
+  as: ipi-conf-aws
+  from: centos:7
+  commands: ipi-conf-aws-commands.sh
+  resources:
+    requests:
+      cpu: 1000m
```

Comment:
Really?

Reply:
> Really?

No idea; blindly copied from [master][1].  Do you have a suggested alternative for all of these read a little, run some Bash, write a little containers?

[1]: https://github.com/openshift/release/blob/be3107193ae1f893a440ec19f78f0151559f00b8/ci-operator/step-registry/ipi/conf/ipi-conf-ref.yaml#L7

Reply:
Yeah let's do `10m`

Reply:
Spun the pivot to 10m out into #7650.  I'll rebase this PR on top once that lands.